### ISSUE TITLE
Synapse is sending vastly different presence status in quick succession

### ISSUE THREAD

**Issue by turt2live** [matrix-org/synapse#3663]
While logging all EDUs coming into a lower traffic server, the following (and similar) got logged quite often:
```
2018-08-07 14:26:35,944 - synapse.federation.transport.server - 294 - INFO - PUT-527- EDU: {u'content': {u'push': [{u'user_id': u'@redacted:domain.com', u'presence': u'online', u'last_active_ago': 801, u'currently_active': True}]}, u'edu_type': u'm.presence'}
2018-08-07 14:26:36,329 - synapse.federation.transport.server - 294 - INFO - PUT-528- EDU: {u'content': {u'push': [{u'user_id': u'@redacted:domain.com', u'presence': u'online', u'last_active_ago': 6975, u'currently_active': True}]}, u'edu_type': u'm.presence'}
```

These appear in two separate transactions, very close together. Shouldn't `last_active_ago` be pretty much the same in the above?

**Comment by neilisfragile** [comment]
related to #3652 ?

**Comment by turt2live** [comment]
Sorry, just noticed your comment. Related in which way?

**Comment by richvdh** [comment]
I doubt #3652 is particularly relevant; however it is related to #3962 in that it's "synapse spews out nonsense about presence"

**Comment by richvdh** [comment]
I reckon this was probably fixed by #4942.
